node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -229,11 +229,8 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
         self.pending_vertex_timeout
     }
 
-    pub(crate) fn drop_dependent_vertices(
-        &mut self,
-        mut vertices: Vec<Dependency<C>>,
-    ) -> HashSet<I> {
-        let mut senders = HashSet::new();
+    pub(crate) fn drop_dependent_vertices(&mut self, mut vertices: Vec<Dependency<C>>) -> Vec<I> {
```

Comment:
Can you add a comment to this method? It accepts a vector of vertices but returns a vector of peers. I actually forgot why and it's not obvious from the method body itself.